spec/unit/module_spec.rb

```diff
@@ -3,20 +3,36 @@
 
 describe R10K::Module do
   describe 'delegating to R10K::Module::Git' do
-    it "accepts args {:git => 'git url}" do
-      obj = R10K::Module.new('foo', '/modulepath', :git => 'git url')
-      expect(obj).to be_a_kind_of(R10K::Module::Git)
+    [
+      ['foo', {git: 'git url'}],
+      ['foo', {type: 'git', source: 'git url'}],
+    ].each do |scenario|
```

Comment:
If you made this an array of maps, you could refer to the name and args separately, instead of using array indices.

Reply:
True. Adding a minor re-work of these tests.